### ISSUE TITLE
Ipe 7.2.8 crashes when interacting with Layers

### ISSUE THREAD

**Issue by thobl** [otfried/ipe#208]
When interacting with the layers via the panel on the left, ipe crashes (e.g., when selecting a layer or when making a layer invisible). The terminal output looks like this:

```
/usr/share/ipe/7.2.8/lua/actions.lua:407: bad argument #2 to 'visible' (layer does not exist)
stack traceback:
	[C]: in method 'visible'
	/usr/share/ipe/7.2.8/lua/actions.lua:407: in local 'f'
	/usr/share/ipe/7.2.8/lua/actions.lua:162: in function </usr/share/ipe/7.2.8/lua/actions.lua:148>
	[C]: in function 'mainloop'
	/usr/share/ipe/7.2.8/lua/main.lua:467: in main chunk
	[C]: in function 'require'
	[string "require "main""]:1: in main chunk
```
Or like this:
```
/usr/share/ipe/7.2.8/lua/actions.lua:490: bad argument #1 to 'isLocked' (layer does not exist)
stack traceback:
	[C]: in method 'isLocked'
	/usr/share/ipe/7.2.8/lua/actions.lua:490: in local 'f'
	/usr/share/ipe/7.2.8/lua/actions.lua:162: in function </usr/share/ipe/7.2.8/lua/actions.lua:148>
	[C]: in function 'mainloop'
	/usr/share/ipe/7.2.8/lua/main.lua:467: in main chunk
	[C]: in function 'require'
	[string "require "main""]:1: in main chunk
```
I am working on arch-linux (installed ipe via aur). A colleague on macOS confirmed that he has the same issue.

**Comment by Willem3141** [comment]
Same here on Linux, but on Windows it seems to work fine.

The cause seems to be that it tries to search for the layer `alpha (0)`, including the `(0)`, which then doesn't exist.

**Comment by otfried** [comment]
Ouch.   

Yes, on Linux and OSX layer actions simply take the text from the layer box as the layer name, which breaks now.

I'll release a new version tomorrow.

**Comment by otfried** [comment]
Fixed in 7.2.9.
